examples/spinner/go.mod

```diff
@@ -0,0 +1,103 @@
+module github.com/ignite/apps/examples/spinner
+
+go 1.23.6
+
+toolchain go1.24.0
```

Comment:
```suggestion
```

Let's remove this, we do not need to require a go 1.24 toolchain (it breaks cosmos sdk chains currently)